gwas_sumstats_tools/interfaces/metadata.py

```
@@ -148,155 +148,130 @@ def metadata_dict_from_gwas_cat(
+    if not is_bypass_rest_api:
+        rest_study_response = download_with_requests(url=rest_study_url)
+        if rest_study_response:  # Add this check
+            try:
+                print(f"{rest_study_url} returned 200")
+                rest_study_dict = _parse_gwas_rest_study_response(
+                    rest_study_response,
+                    replace_dict=REST_API_STUDY_MAPPINGS,
+                    fields_to_split=STUDY_FIELD_TO_SPLIT,
+                )
+                meta_dict.update(rest_study_dict)
+            except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)